server/game/core/zone/ConcreteArenaZone.ts

```diff
@@ -0,0 +1,72 @@
+import { InPlayCard } from '../card/baseClasses/InPlayCard';
+import { Card } from '../card/Card';
+import { ZoneName } from '../Constants';
+import Game from '../Game';
+import Player from '../Player';
+import * as Contract from '../utils/Contract';
+import { ConcreteOrMetaArenaZone, IArenaZoneCardFilterProperties } from './ConcreteOrMetaArenaZone';
+import { IAddRemoveZone } from './ZoneAbstract';
+
+export abstract class ConcreteArenaZone extends ConcreteOrMetaArenaZone implements IAddRemoveZone {
```

Comment:
Can we add some docs on each of these classes describing what they hold/do? Doesn't have to be super in-depth, but just to explain the hierarchy a bit